exotica_examples/scripts/example_sparseddp_valkyrie

```diff
@@ -9,121 +9,37 @@ from visualization_msgs.msg import Marker, MarkerArray
 
 import numpy as np
 import pyexotica as exo
-import exotica_ddp_solver_py
 from pyexotica.publish_trajectory import *
 
+import sys
 
 from collections import OrderedDict
 import matplotlib.pyplot as plt
 
-def task_cost_over_time(problem):
```

Comment:
Maybe preserve this and move it to pyexotica/tools or plot --> it's useful to have this for generic analysis